deps/build.jl

```diff
@@ -0,0 +1,28 @@
+using BinDeps
+using Compat
+
+@BinDeps.setup
+
+libmysqlclient = library_dependency("libmysqlclient",
+                                    alias=["libmysqlclient.so",
```

Comment:
I don't know the answer to that question specifically, but I did make a `usr` folder and manually move the contents of the extracted zip (`bin`, `include`) etc, and then `Pkg.build` worked and generated a `deps.jl` file.